dashboard/frontend/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "tensorflowk8s-dashboard",
   "version": "0.1.0",
   "description": "Dashboard for kubeflow/tf-operator.",
+  "homepage": ".",
```

Comment:
Homepage defines where the frontend will query for the static assets. If not specified the static assets will be queried at the root, i.e: `127.0.0.1:8001/static` which breaks.
`"."` means that we query at the path where the application is served. i.e: `http://127.0.0.1:8001/api/v1/namespaces/kubeflow/services/ambassador:80/proxy/tfjobs/ui/static`

Reply:
Please make this a comment in the code.